### PR TITLE
Update associated files for JetBrains products

### PR DESCRIPTION
**Pull request by ghost**
Add/Update associated files for Intellij-idea, PhpStorm, PyCharm, RubyMine, DataGrip, and fix previous wrong paths in rubymine.rb

### CODE REVIEW

**Comment by vitorgalvao** [comment]
In the future, please submit separate pull requests for each cask, as more often than not this type of pull request brings problems with one or more of them, and it halts the inclusion of the other ones.

Also, would you mind doing this for the ones in [caskroom/versions](https://github.com/caskroom/homebrew-versions)?

File: Casks/rubymine.rb [hunk 1]
```diff
@@ -10,7 +10,11 @@
   app 'RubyMine.app'
 
   zap delete: [
-                "~/Library/Application Support/RubyMine#{version.delete('.')}",
-                "~/Library/Preferences/RubyMine#{version.delete('.')}",
+                '~/Library/Application Support/RubyMine80',
```
Comment (vitorgalvao) [inline Casks/rubymine.rb]: Use `#{version.major_minor.no_dots}` instead of `80`.

File: Casks/pycharm.rb [hunk 2]
```diff
@@ -8,4 +8,13 @@
   license :commercial
 
   app 'PyCharm.app'
+
+  zap delete: [
+                '~/.PyCharm50',
```
Comment (vitorgalvao) [inline Casks/pycharm.rb]: Use `#{version.major_minor.no_dots}` instead of `50`.

File: Casks/phpstorm.rb [hunk 3]
```diff
@@ -10,6 +10,9 @@
   app 'PhpStorm.app'
 
   zap delete: [
+                '~/.WebIde100',
```
Comment (vitorgalvao) [inline Casks/phpstorm.rb]: Use `#{version.major_minor.no_dots}` instead of `100`.

File: Casks/intellij-idea.rb [hunk 4]
```diff
@@ -10,6 +10,9 @@
   app 'IntelliJ IDEA 15.app'
 
   zap delete: [
+                '~/.IntelliJIdea15',
```
Comment (vitorgalvao) [inline Casks/intellij-idea.rb]: Use `#{version.major}` instead of `15`.

File: Casks/datagrip.rb [hunk 5]
```diff
@@ -8,4 +8,13 @@
   license :commercial
 
   app 'DataGrip.app'
+
+  zap delete: [
+                '~/.DataGrip10',
```
Comment (vitorgalvao) [inline Casks/datagrip.rb]: Use `#{version.major_minor.no_dots}` instead of `10`.

**Comment by gksu** [comment]
@victorpopkov Thanks for your reminder

**Comment by gksu** [comment]
@vitorgalvao I will do it these days, when I got time

**Comment by vitorgalvao** [comment]
Thank you!

**Pull request merged**